apps/api/views_drf.py

```diff
@@ -474,3 +477,24 @@ def delete(self, request, addon_id, version_id):
         version = self.get_object()
         version.delete()
         return Response(status=204)
+
+
+class GithubView(APIView):
+
+    def post(self, request):
+        if request.META.get('HTTP_X_GITHUB_EVENT') != 'pull_request':
+            # That's ok, we are just going to ignore it, we'll return a 2xx
+            # response so github doesn't report it as an error.
+            return Response({}, status=202)
```

Comment:
The first bit is "The request has been accepted for processing" though and at this point we already know that we're done processing. So it seems weird to me. It really doesn't matter though. Should only be computers seeing it.